test/support/components/layouts_jetpack/app.swiftui.neex

```diff
@@ -0,0 +1 @@
+<AppContainer><%= @inner_content %></AppContainer>
```

Comment:
Does this swiftui file must be included?

Reply:
fixed, that was a missed file rename